core/tools/norm_unicode_update.cpp

```diff
@@ -0,0 +1,84 @@
+#include "kmx/kmx_plus.h"
+#include "kmx/kmx_xstring.h"
+#include "core_icu.h"
+
+#include <fstream>
+#include <iostream>
+#include <string>
+#include <vector>
+
+#include <assert.h>
+
+#include <unicode/uchar.h>
+
+#ifndef __EMSCRIPTEN__
+
+int
+write_nfd_table(const char *NFD_FILE) {
+  std::cout << " writing: " << NFD_FILE << std::endl;
+  auto f =  std::ofstream(NFD_FILE);
+  assert(f.good());
+
+  // write preamble
+  f << "//NFD hasBoundaryBefore" << std::endl;
```

Comment:
This needs to say what tool is generating this file and how to regenerate it.

Reply:
100%… trying to get it to function first.